src/logic/lint.ts

```diff
@@ -301,7 +302,7 @@ async function prepareLintedProfile(
     try {
       ast = parseProfile(new Source(profileSource.source, profileSource.path));
     } catch (e) {
-      report.errors.push(e);
+      report.errors.push(e as SyntaxError | ValidationIssue);
```

Comment:
nitpick(not-blocking): this isn't really type safe, but I assume the errors are then just printed right?

Reply:
It isn't safe. Best solution would be use some guards but I want to do it in separate PR.

Reply:
I opened issue for that :)

Reply:
🙏